doc/cookbook/source/examples/classifier/mkl_binary_class.rst

```diff
@@ -0,0 +1,59 @@
+==================================================
+Multiple Kernel Learning for Binary Classification
+==================================================
+
+Multiple kernel learning (MKL) is about using a combined kernel i.e. a kernel consisting of a linear combination of arbitrary kernels over different domains.
+
+.. math::
+
+    f({\bf x})=\text{sign} \left(\sum_{i=0}^{N-1} \alpha_i k({\bf x}, {\bf x_i})+b\right)
```

Comment:
Actually, I would not give this line here, this is already mentioned in the kernel svm cookbook (you could reference that like @OXPHOS did in #3242 ).
Rather the math line should be the k=\sum_i k_i .....